PeerioTechnologies/peerio-icebear
# fix: invite accepted event

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/6764/desktop-contact-accepted-notification-snackbar-tweak

#### Testing instructions  
1. when you add or remove contacts from favourites, snackbar should no longer be displayed
2. when contact accepts invite, they shouldn't be added to favourites by default
3. PendingDM actually broke the desktop notification for accepted invites and we didn't notice. verify that these are working again. (invite new user -> new user signs up and confirms email -> original user sees desktop notification)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
